lib/hawkular/base_client.rb

```diff
@@ -175,6 +178,16 @@ def tenant_header
       headers
     end
 
+    # @!visibility private
```

Comment:
This is something that I copy/pasted, I guess that doesn't do anything. I will remove if not needed.